server.py

```diff
@@ -211,8 +211,9 @@ def create_model_menus():
                 ui.create_refresh_button(shared.gradio['model_menu'], lambda: None, lambda: {'choices': get_available_models()}, 'refresh-button')
         with gr.Column():
             with gr.Row():
-                shared.gradio['lora_menu'] = gr.Dropdown(choices=available_loras, value=shared.lora_name, label='LoRA')
+                shared.gradio['lora_menu'] = gr.CheckboxGroup(choices=available_loras, value=shared.lora_names, label='LoRA model(s)')
```

Comment:
I don't know how many LoRAs people might end up with, but you could maybe keep this a `Dropdown` and add the `multiselect=True` argument. Probably a clearer UI experience?

Reply:
Oh wow, I'm dumb. In #853 this is what the auto webui used for `styles` (my suggested option #4) but I never looked into how that was done. This is indeed much cleaner. Going to change that and push, though it seems I'll need to rebase and force-push as Main branch changed code right next to the server.py edits here.